src/desktop/components/branching_state/test/index.test.coffee

```diff
@@ -9,7 +9,7 @@ describe 'State', ->
       state.context.state.should.equal state
 
     describe '#inject', ->
-      it 'lets you inject additions to the context at any point', ->
+      xit 'lets you inject additions to the context at any point', ->
```

Comment:
Assuming this is deliberate?

Reply:
Yeah this one was broken -- maybe wasn't running before?